cli/tink/cmd/workflow/create.go

```diff
@@ -40,14 +35,14 @@ var createCmd = &cobra.Command{
 func addFlags() {
 	flags := createCmd.PersistentFlags()
 	flags.StringVarP(&template, "template", "t", "", "workflow template")
-	flags.StringVarP(&target, "target", "r", "", "workflow target")
+	flags.StringVarP(&hardware, "hardware", "r", "", "workflow target hardwares")
 
-	createCmd.MarkPersistentFlagRequired(fTarget)
+	createCmd.MarkPersistentFlagRequired(fHardware)
 	createCmd.MarkPersistentFlagRequired(fTemplate)
 }
 
 func createWorkflow(c *cobra.Command, args []string) {
-	req := workflow.CreateRequest{Template: template, Target: target}
+	req := workflow.CreateRequest{Template: template, Target: hardware}
```

Comment:
Please update the proto, such that `Target` is `Hardware`.

Reply:
I don't this it is required because `hardware` is also a kind of target so I have changed the name and type of `target` column in the `workflow` table to `devices` and `JSONB`.